crates/wallet/tests/wallet.rs

```diff
@@ -956,27 +956,6 @@ fn test_create_tx_drain_to_dust_amount() {
     builder.finish().unwrap();
 }
 
-#[test]
-fn test_create_tx_ordering_respected() {
```

Comment:
🔧 rather than removing this test can you just copy/paste the code for the `custom_bip69_ordering` and use that here? I don't want to lose the coverage this tests give us that confirms the `TxBuilder` uses the sorting you give it.